setup.py

```diff
@@ -34,7 +37,7 @@
 
 install_requires = [
     "apispec>=0.21.0,<0.40",
-    "black>=19.10b0,<20",
+    "black>=22",
```

Comment:
Note: relying on `black` greater than 22 (no longer pre-release!) which should be OK for Python 3.6.  Keeping upper pin opened for the future, hoping the black formatted output won't be changing much with newer versions.